### PR TITLE
LINK-1351, part 1 | Registration admin selection

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Adds the possibility to select registration admins for an organization using the Django admin site and the API endpoint òrganization` (`OrganizationViewSet`). We wanted the field to appear in the same way as the other similar fields for selecting regular admins and users which is why a custom form has been used instead of using the built-in "inlines" functionality of Django admin.

This PR is the first one of two PRs - the other PR (part 2) will bring the permission changes described in the Jira task.
### Partially closes
[LINK-1351](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1351)

[LINK-1351]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by voneiden** [comment]
The PR has 30 commits from Jori that are (probably?) already merged to `main` , rebase please?

**Comment by harriris-vincit** [comment]
@voneiden Rebased to `main` now

File: events/tests/test_organization_api.py [hunk 1]
```diff
@@ -272,6 +270,31 @@ def test_admin_user_add_users_to_new_organization(
...
+
+    response = assert_create_organization(user_api_client, payload)
+    assert set(payload["admin_users"]) == set(
+        [i["username"] for i in response.data["admin_users"]]
+    )
+    assert set(payload["registration_admin_users"]) == set(
+        [i["username"] for i in response.data["registration_admin_users"]]
+    )
+
+
```
Comment (tuomas777) [inline events/tests/test_organization_api.py:297]: I think it is better when these tests also check the database just to be sure, as is done in the other new test `test_admin_user_update_organization_registration_admin_users()`.

**Review by voneiden — COMMENTED** [review]
I had this review partially done, but since Tuomas has managed to finish it I'll just leave the note I had pending here. Overall LGTM.

File: events/tests/test_organization_api.py (hunk 1 above)
Comment (harriris-vincit) [reply events/tests/test_organization_api.py:304]: Changed as suggested now

File: helevents/admin.py [hunk 2]
```diff
@@ -1,9 +1,43 @@
...
+            form.user_admin_site = user_modeladmin.admin_site
+        form.wrapper_kwargs = wrapper_kwargs
+
+        return form
+
+    def save_model(self, request, obj, form, change):
+        registration_admin_users = form.cleaned_data.get("registration_admin_users")
+        obj.registration_admin_users.clear()
+        if registration_admin_users:
+            obj.registration_admin_users.set(registration_admin_users)
```
[the remark this answers is not in the document]
Comment (harriris-vincit) [reply helevents/admin.py]: Changed as suggested now.

**Comment by harriris-vincit** [comment]
@tuomas777 &  @voneiden: I've Implemented the suggestions left by Tuomas - re-requesting review :-)

**Review by voneiden — APPROVED** [review]
:shipit:

**Pull request merged**